crates/binstalk/src/ops/resolve.rs

```diff
@@ -267,17 +267,22 @@ async fn download_extract_and_verify(
                 // This binary is optional
                 Err(err) => {
                     let required_features = &bin.required_features;
+                    let bin_name = bin.name.as_str();
 
                     if required_features.is_empty() {
+                        error!(
+                            "When resolving {name} bin {bin_name} is not found.\
+This binary is not optional so it must be included in the achieve, please contact with\
```

Comment:
archive

Reply:
```suggestion
This binary is not optional so it must be included in the archive, please contact with\
```

Reply:
P.S. having a siggested change would make it easier to update the branch